addon/components/grid-stack.js

```diff
@@ -151,7 +150,7 @@ export default class GridStackComponent extends Component {
 
```

Comment:
Can you update the documentation for this method to point at https://github.com/gridstack/gridstack.js/blob/v7.2.2/src/gridstack.ts#L1052 instead and it looks like we can update this block
```js
// The following line was causing issues because this hook is called correctly from
// child widgets, but after they are already removed from the dom
// --- SKIP ---
// if (el.parentElement !== this.el) return; // not our child!
// --- SKIP ---
```

to this now

```js
if (el.parentElement && el.parentElement !== this.el) return; // not our child!
```

Reply:
Sure, I've also updated the block, seems to work fine.

Reply:
Actually now that I look at it again, the only custom part was that one commented out line which now works, so we should be able to replace the whole method with just
```js
triggerEvent = triggerEvent && !this.isDestroying && !this.isDestroyed;
this.gridStack?.removeWidget(element, removeDOM, triggerEvent);
```

Sorry for not noticing that sooner and thank you for the update!

Reply:
Yeah, you' re right, I probably should then remove that part too, right?
  /**
   * Custom removeWidget function that skips check to see if widget is in current grid
   * @see https://github.com/gridstack/gridstack.js/blob/v7.2.2/src/gridstack.ts#L1052
   */

Reply:
Yep!